bundles/org.openhab.binding.x/README.md

```diff
@@ -1,16 +1,16 @@
-# Twitter Binding
+# X Binding
 
-The Twitter binding allows your home to Tweet 280 characters at a time. It also supports direct messages and tweeting with media.
+The X (formerly known as Twitter) binding allows your home to Tweet 280 characters at a time. It also supports direct messages and tweeting with media.
```

Comment:
As this PR is a breaking change you might as well replace "tweet" with "post" everywhere now too.

Reply:
Done.